pkg/wsman/cim/models/types.go

```diff
@@ -375,18 +375,23 @@ type ServiceAvailableToElement struct {
 
 type AssociationReference struct {
 	Address             string `xml:"Address,omitempty"`
-	ReferenceParameters ReferenceParmetersNoNamespace
+	ReferenceParameters ReferenceParametersNoNamespace
 }
 
-type ReferenceParmetersNoNamespace struct {
-	ResourceURI string                `xml:"ResourceURI,omitempty"`
-	SelectorSet []SelectorNoNamespace `xml:"SelectorSet>Selector,omitempty"`
+type ReferenceParametersNoNamespace struct {
+	XMLName     xml.Name            `xml:"ReferenceParameters,omitempty"`
+	ResourceURI string              `xml:"ResourceURI,omitempty"`
+	SelectorSet SelectorNoNamespace `xml:"SelectorSet,omitempty"`
```

Comment:
Was the breaking out of `SelectorSet>Selector` necessary? Or was that just a part of testing? In other words, do we really need the two structs SelectorNoNamespace and SelectorResponse ?